frontend/src/components/modules/Tournaments/OngoingTournament/RoundRobinTournamentView.tsx

```diff
@@ -153,81 +142,28 @@ const RoundRobinTournamentView: React.FC = () => {
   const sortMatches = (): void => {
     const matches: Match[] = tournament.matchSchedule;
     if (matches.length > 0) {
-      const sortedMatches = matches
-        .filter((match) => match.elapsedTime !== undefined)
-        .sort((a, b) => {
-          return b.elapsedTime - a.elapsedTime;
-        });
-      ongoingMatches = sortedMatches.filter(
-        (match) =>
-          match.elapsedTime !== null &&
-          match.elapsedTime > 0 &&
-          match.winner === undefined
+      const sortedMatches = matches.sort((a, b) => {
+        return b.elapsedTime - a.elapsedTime;
+      });
+
+      setOngoingMatches(
+        sortedMatches.filter(
+          (match) => match.elapsedTime > 0 && match.winner === undefined
+        )
       );
-      upcomingMatches = sortedMatches.filter(
-        (match) =>
-          (match.elapsedTime === null || match.elapsedTime <= 0) &&
-          match.winner === undefined
+      setUpcomingMatches(
+        sortedMatches.filter(
+          (match) => match.elapsedTime <= 0 && match.winner === undefined
+        )
       );
-      pastMatches = sortedMatches.filter(
-        (match) =>
-          match.elapsedTime !== null &&
-          match.elapsedTime > 0 &&
-          match.winner !== undefined
+      setPastMatches(
+        sortedMatches.filter(
+          (match) => match.elapsedTime > 0 && match.winner !== undefined
+        )
```

Comment:
Afaik, the elapsedTime will never be undefined (at least according to it's type and my memory). I also went ahead and tested to see if this is true, but at least in those cases they always had an underying value